app/src/helpers/helpers.ts

```diff
@@ -185,3 +185,8 @@ export function removeUserAgentSpecifics(
     .replace(`Electron/${process.versions.electron} `, '')
     .replace(`${appName}/${appVersion} `, ' ');
 }
+
+// Removes formatting from a text
+export function getPlainTextWithoutFormatting(text: string): string {
+  return text.trim().replace(/\n/g, ' ').replace(/\s+/g, ' ');
```

Comment:
As discussed above, not sure about stripping newlines.

```suggestion
  return text.trim().replace(/\s+/g, ' ');
```

?

Reply:
Ok sure. I will do all the requested changes. Thanks for the detailed review!